Added toolbars to all activities and Danish localisation

agraven opened the pull request:
Also made some minor theming tweaks, which still need more work. Have a look at the android doc Material design pages.

Localisation was implemented by switching to string resources instead of hardcoded strings.

Also, I spint way to long going through lint complaints, and there's still so many left...

TODO: Skal vi bruge mit eller dit fork som hoved repositorie?

agraven committed:
Added toolbars to all activities and Danish localisation

Also made some minor theming tweaks, which still need more work. Have a look at the android doc Material design pages.

Localisation was implemented by switching to string resources instead of hardcoded strings.

Also, I spint way to long going through lint complaints, and there's still so many left...

theodor349 reviewed (approved):
Skal vi ikke bare bruge min branch. Og så kan jeg bare lave en ny som jeg editor fra. Så vi har en main og en vi hver i sær bruger. Og så kan vi push til main når vi har noget der virker?

agraven commented:
Ok, det lyder som en smart plan, så gør vi det. Jeg har en rimelig bredt
dækkende pull request jeg snart sender fordi jeg har været ved at gå alle
advarslerne lint har råbt over. Der er skræmmende mange. Så har jeg også
tænkt mig at være lidt mere vedholdende ved tematisering af vores ting, og
definere et tema vores app bruger.

agraven committed:
Even more lint fixes

A whole bunch of lint complaints were fixed. Some still remain, but are of lower priority.

theodor349 reviewed (approved):
Jo det syntes jeg lyder fint. Jeg kan begynde når dy har fået de sidste ting på plads. Så du må godt lige sige til når det er gjort 👍
